### ISSUE TITLE
Parallelise postage stamps

### ISSUE THREAD

**Issue by ajstewart** [ajstewart/racs-historical-transients#3]
May be possible to produce these faster by parallelising the code, but may run into issues of an image attempting to be accessed at the same time. In addition to multiple versions of the same plot being loaded into memory.

**Comment by ajstewart** [comment]
This has been attempted but I could not get `matplotlib` to play nice when generating the plots in parallel. Text would sometimes appear misplaced or stretched, including the legend. I think the issue is rooted at savefig or something to do with how `matplotlib` adds text to the figure. 

See examples below. For now on hold. Probably best to run multiple instances of the process at once to get through images.

Separate plotting script?

![sumss_21h08m43 01s_-48d55m02 2001s_sidebyside](https://user-images.githubusercontent.com/3306161/51960424-131b2900-24ad-11e9-91ce-8f1827af8d1e.jpg)
![sumss_20h26m46 47s_-50d18m43 8998s_sidebyside](https://user-images.githubusercontent.com/3306161/51960428-17dfdd00-24ad-11e9-8759-9349e997bf2c.jpg)

**Comment by ajstewart** [comment]
This will be possible when dropping aplpy and switching to matplotlib/astropy. A new postage stamp routine will probably be needed.
